presto-main/src/main/java/io/prestosql/testing/LocalQueryRunner.java

```diff
@@ -848,7 +848,7 @@ public Plan createPlan(Session session, @Language("SQL") String sql, List<PlanOp
         LogicalPlanner logicalPlanner = new LogicalPlanner(session, optimizers, new PlanSanityChecker(true), idAllocator, metadata, new TypeAnalyzer(sqlParser, metadata), statsCalculator, costCalculator, warningCollector);
 
         Analysis analysis = analyzer.analyze(preparedQuery.getStatement());
-        return logicalPlanner.plan(analysis, stage);
+        return logicalPlanner.plan(analysis, stage, true);
```

Comment:
`SystemSessionProperties.isCollectPlanStatisticsForAllQueries(session)`

Reply:
Is there any reason to disregard `session` configuration in LQR?
If it's done purposefully, please add explanatory comment.

Reply:
added comment

Reply:
what happens if you have `false` or `isCollectPlanStatisticsForAllQueries(session)` here?

according to your change's goals, tests should run faster this way, so why not?